Core/src/org/sleuthkit/autopsy/discovery/ui/MiniTimelineDateListPanel.java

```diff
@@ -0,0 +1,242 @@
+/*
```

Comment:
It looks like this file was generated with the gui designer.  Is there are corresponding .form file?

Reply:
Same as the other list no form file because it was a refactored copy